scipy/stats/_pagel.py

```diff
@@ -0,0 +1,474 @@
+from itertools import permutations
+import numpy as np
+from ._continuous_distns import norm
+import scipy.stats
+from dataclasses import make_dataclass
+
+
+PageLResult = make_dataclass("PageLResult", ("statistic", "pvalue", "method"))
+
+
+def pagel(data, ranked=True, predicted_ranks=None, method='auto'):
+    r"""
+    Compute Page's L, a measure of trend in observations between treatments.
+
+    Page's L is useful when:
+
+    * there are :math:`n \geq 3` treatments,
+    * :math:`m \geq 2` subjects are observed for each treatment, and
+    * the observations are hypothesized to have a particular order.
+
+    Specifically, the test considers the null hypothesis that
+
+    .. math::
+
+        m_1 = m_2 = m_3 \cdots = m_n,
+
+    where :math:`m_j` is the mean of the observed quantity under treatment
+    :math:`j`, against the alternative hypothesis that
+
+    .. math::
+
+        m_1 < m_2 < m_3 < \cdots < m_n.
```

Comment:
We should state the alternative as it is stated in Odeh, and in Weil & Di Bucchianico [0]_:
```suggestion
        m_1 ≤ m_2 ≤ m_3 ≤ \cdots ≤ m_n,

    with at least one strict inequality.
```
I used the Unicode character `≤`.  If you keep it, that character will have to be added to `allowed` in https://github.com/scipy/scipy/blob/master/tools/unicode-check.py.

[0] Weil, van de, M. A, & Di Bucchianico, A. (1998), Fast computation of the exact null distribution of Spearman's rho and Page's L statistic for samples with and without ties, (Memorandum COSOR; Vol. 9817). Technische Universiteit Eindhoven.

Reply:
I used `\leq` since we're int he middle of a bunch of LaTeX anyway.